UnityMcpBridge/UnityMcpServer~/src/tools/manage_gameobject.py

```
@@ -1,87 +1,73 @@
-        parent: str = None,  # Used for both 'create' (initial parent) and 'modify' (change parent)
-        position: List[float] = None,
-        rotation: List[float] = None,
-        scale: List[float] = None,
-        components_to_add: List[str] = None,  # List of component names to add
-        primitive_type: str = None,
-        save_as_prefab: bool = False,
-        prefab_path: str = None,
-        prefab_folder: str = "Assets/Prefabs",
+        ctx: Context,
+        action: Literal["create", "modify", "delete", "find", "add_component", "remove_component", "set_component_property", "get_components"],
+        target: Annotated[str,
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)